# deep learning papers

This is a collection of papers on "deep learning" related to computational biology.
The point of this repository is to get papers that others think are important or translate well to problems in computational biology.
I hope to pick a subset of these for our "deep learning for comp bio" reading group in Spring 2016.

In particular, my interests lie in:

- dimension reduction techniques
- clustering techniques
- functional annotation
- other things that I probably don't know about that are cool

## reviews

- [Deep learning](http://www.nature.com/nature/journal/v521/n7553/full/nature14539.html)

## blogs

- [colah's blog](http://colah.github.io/)

## functional annotation

- [ADAGE analysis of publicly available gene expression data collections illuminates Pseudomonas aeruginosa-host interactions](http://biorxiv.org/content/early/2015/11/05/030650)
- [Basset: Learning the regulatory code of the accessible genome with deep convolutional neural networks.](http://biorxiv.org/content/early/2015/10/05/028399)
- [The human splicing code reveals new insights into the genetic determinants of disease](http://www.sciencemag.org/content/347/6218/1254806.long)
- [Unsupervised feature construction and knowledge extraction from genome-wide assays of breast cancer with denoising autoencoders](http://www.worldscientific.com/doi/abs/10.1142/9789814644730_0014)

## theory

- [A Probabilistic Theory of Deep Learning](http://arxiv.org/abs/1504.00641)
- [Autoencoders, Unsupervised Learning, and Deep Architectures](http://jmlr.csail.mit.edu/proceedings/papers/v27/baldi12a/baldi12a.pdf)
- [Deep rectifier sparse neural networks](http://jmlr.org/proceedings/papers/v15/glorot11a/glorot11a.pdf)
- [Deep Learning with Nonparametric Clustering](http://arxiv.org/abs/1501.03084)
- [k-Sparse Autoencoders](http://arxiv.org/pdf/1312.5663v2.pdf)
